docs/data/run-frontier-squid-container.md

```diff
@@ -0,0 +1,133 @@
+Running Frontier Squid in a Container
+=====================================
+
+Frontier Squid is a distribution of the well-known [squid HTTP caching
+proxy software](http://squid-cache.org) that is optimized for use with
+applications on the Worldwide LHC Computing Grid (WLCG). It has
+[many advantages](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Why_use_frontier_squid_instead_o)
+over regular squid for common grid applications, especially Frontier
+and CVMFS. The OSG distribution of frontier-squid is a straight rebuild of the
+upstream frontier-squid package for the convenience of OSG users.
+
+!!! tip
+    OSG recommends that all sites run a caching proxy for HTTP and HTTPS to help reduce bandwidth and improve
+    throughput.
+
+This document outlines how to run Frontier Squid in a Docker container.
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points (consulting [the Reference section below](#reference) as needed):
+
+1. **Docker:** For the purpose of this guide, the host must have a running docker service
+   and you must have the ability to start containers (i.e., belong to the `docker` Unix group).
+1.   **Network ports:** Frontier squid communicates on ports 3128 (TCP) and 3401 (UDP)
```

Comment:
While we're at it, let's expand on this and the same line in https://opensciencegrid.org/docs/data/frontier-squid/ using guidance from Dave's response here https://support.opensciencegrid.org/public/tickets/fc3a0dc0ab486c69fbf40e976eab6d54ecad10970785635586321e56c2c4c579

Reply:
@brianhlin, i suspect you are referring to this comment from Dave:

>Next, generally people do not filter the outgoing connections from squid, but cvmfs always uses ports 8000, 80, or 8080.   If in the future you allow CMS or ATLAS to run jobs at your site, they will use the Frontier application which also normally uses port 8000, although ATLAS does have some odd ports used by some of their Frontier servers.
>
>Finally, we really encourage sites to allow monitoring on port 3401 via UDP from CERN.  The OSG frontier-squid container does support that.  Usually it also requires some enabling from iptables and/or site firewalls.  If it cannot be done it is not a showstopper, but it really helps with diagnosing problems.  The details are at
>    https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Enabling_monitoring

What specifically would you like to incorporate into the Before Starting section?  Just the 80/8000/8080 port numbers?

Reply:
Alright updated, hopefully to better answer Dan's questions